src/Config/Events.php

```diff
@@ -3,7 +3,11 @@
 use CodeIgniter\Events\Events;
 use Config\Services;
 
-Events::on('post_controller_constructor', function () {
+Events::on('post_controller_constructor', function () { 	
+	
+	$config = config('Tatter\Visits\Config\Visits');
```

Comment:
Shorthand Config names are preferable:
```suggestion
	$config = config('Visits');
```